src/server/document-input-controller.js

```diff
@@ -69,6 +107,13 @@ class DocumentInputController {
     return res.send({});
   }
 
+  registerPages(router) {
+    router.get(
+      '/doc-inputs/:documentInputId*',
```

Comment:
`documentInputId` is just a string without any sub paths, so I think we don't need the asterisk here.

Reply:
Corrected 👍